core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -94,6 +98,16 @@ public static void setUpStatic() {
         } catch (IOException e) {
             throw new ExceptionInInitializerError();
         }
+
+        hideTelemetryTable = false;
```

Comment:
nit: other static fields are being reset in `tearDown`, so it might be a good idea to move this one there for the sake of consistency.

Reply:
good idea, done here: https://github.com/questdb/questdb/pull/1951/commits/055fe50d49603384f583b83dbcd506eda98d4940